packages/sources/coinpaprika-test/src/config/index.ts

```diff
@@ -0,0 +1,10 @@
+export const DEFAULT_API_ENDPOINT = 'https://api.coinpaprika.com'
+export const PRO_API_ENDPOINT = 'https://api-pro.coinpaprika.com'
+
+export const customSettings = {
```

Comment:
I also noticed `coinpaprika` has some unique functionality based on an env var called `IS_TEST_MODE`. We may need to include that here too unless there was a discussion on not needing this anymore

Reply:
This would also require changes where you prepare requests since you'd have to set a header based on this.